### PR TITLE
Add support for healthz in webpkgserver

### LINKED ISSUE #88: [webpkgserver] add something for health check
Are there any plans to add a health check path to webpkgserver? We want to monitor the webpkgserver and use the k8s livenessProbe.

ex. https://github.com/ampproject/amppackager/pull/359

### PR DESCRIPTION
**Pull request by banaag**
Fixes #88

### CODE REVIEW

File: server/handler.go [hunk 1]
```diff
@@ -182,6 +184,22 @@ func (h *Handler) handleValidity(w http.ResponseWriter, req *http.Request) {
 	replyOK(w, emptyMapCBOR, mimeTypeValidity)
 }
 
+func (h *Handler) handleHealth(w http.ResponseWriter, req *http.Request) {
+	ac := h.CertManager.GetAugmentedChain()
+	if ac == nil {
+		replyError(w, http.StatusNotFound)
+		return
+	}
+	err := ac.VerifyAll(timeutil.Now(), false)
```
Comment (twifkak) [inline server/handler.go:193]: Would it make sense to change `false` to `!AllowTestCert`? https://github.com/google/webpackager/blob/1026e99cbadb70c6967fc2937e50260dbfb83934/cmd/webpkgserver/webpkgserver.example.toml#L200-L209

**Review by twifkak — CHANGES_REQUESTED** [review]

File: server/handler.go (hunk 1 above)
Comment (banaag) [reply server/handler.go]: Yes, updated.

**Review by twifkak — APPROVED** [review]
Thanks!

**Review by twifkak — APPROVED** [review]

**Pull request merged**